projects/legacy/components/input-tag/input-tag.style.less

```diff
@@ -82,6 +82,10 @@
     :host[data-size='l']:not(._label-outside) & {
         padding: 0;
     }
+
+    & > tui-tag:nth-child(1):has(+ .t-input-wrapper) {
```

Comment:
Good idea, but it breaks if there are tags before:
<img width="510" alt="image" src="https://github.com/user-attachments/assets/19a83a32-3fe6-46bf-a8d7-910a5c6da37e">
How about we just make :last-child tag max width 80%?

Reply:
@waterplea fixed